pkg/rancher-desktop/utils/protocols.ts

```diff
@@ -78,11 +78,9 @@ export const protocolsRegistered = Latch();
  */
 function setupAppProtocolHandler() {
   // TODO: #5659 - Deprecated in Electron 25; replace with protocol.handle
-  // eslint-disable-next-line deprecation/deprecation
```

Comment:
Hmm, are you running a different version of `electron`? This deprecation suppression shouldn't be removed yet (the CI runs are failing without it).  I think just undoing the commit that made changes to this file would be fine. (Preferably by rebasing it out rather than doing a revert.)

Reply:
The eslint comment isn't in `main` anymore. Pulling and rebasing fixed the merge conflict.